cmd/aries-agent-rest/startcmd/start.go

```diff
@@ -46,12 +52,25 @@ const (
 	agentTokenFlagUsage     = "Check for bearer token in the authorization header (optional)." +
 		" Alternatively, this can be set with the following environment variable: " + agentTokenEnvKey
 
-	// db path flag
-	agentDBPathFlagName      = "db-path"
-	agentDBPathEnvKey        = "ARIESD_DB_PATH"
-	agentDBPathFlagShorthand = "d"
-	agentDBPathFlagUsage     = "Path to database." +
-		" Alternatively, this can be set with the following environment variable: " + agentDBPathEnvKey
+	databaseTypeFlagName      = "database-type"
```

Comment:
There are many parameters already. Suggest we merge this one into databaseURL in a `type://url` format... see https://github.com/trustbloc/edge-sandbox/blob/4b1f867848ced0394ba37172b919ef4898cebec8/cmd/common/common.go#L40-L46

Reply:
it's better to separate less complexity take look at   https://github.com/trustbloc/edge-service/blob/master/cmd/vc-rest/startcmd/start.go#L96